backend/app/api/routes/inquiries.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+
+from fastapi import APIRouter, HTTPException
+
+from app import crud
+from app.api.deps import SessionDep
+from app.models import Inquiry, InquiryCreate, InquiryPublic
+
+router = APIRouter()
+
+
+@router.post("/", response_model=InquiryPublic)
+def create_inquiry(
+    *, session: SessionDep, inquiry_in: InquiryCreate
+) -> Any:
+    """
+    Create new inquiry.
+    """
+    inquiry = crud.get_inquiry_by_text(session=session, text=inquiry_in.text)
+    if inquiry:
+        raise HTTPException(
+            status_code=400,
+            detail="This question already exists.",
+        )
+
+    inquiry = Inquiry.model_validate(inquiry_in)
```

Comment:
Question -- is there a possible race condition here if the API is hit multiple times with the same data?